IgOmeProfiling_pipeline.py

```diff
@@ -165,6 +172,7 @@ def run_pipeline(fastq_path, barcode2samplename_path, samplename2biologicalcondi
     parser.add_argument('--shuffles', default=5, type=int, help='Number of controlled shuffles permutations')
 
     # general optional parameters
+    parser.add_argument('--type_machines_to_stop', type=str, default='t2.medium_t2.2xlarge_m5a.24xlarge', help='Choose witch type of machine to stop')
```

Comment:
there shouldn't be a default.
Default should be empty meaning stop all machine types.
It also would be better instead or in addition to stop by name pattern, for example "wroker*".

There should also be a flag whether to stop or not and pass it step 3